packages/docs/next-doc-site/components/editor.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-shadow */
+import { Fragment } from 'react';
+import SimpleEditor from 'react-simple-code-editor';
+import { TEditorProps, useValueDebounce } from 'react-view';
```

Comment:
Ooh now I know you follow the casing even when it's uncomfortable. You're stronger than I am